python/ray/tune/utils/log.py

```diff
@@ -0,0 +1,46 @@
+from enum import Enum
+from typing import Any, Callable, Union
+
+
+class Verbosity(Enum):
+    MINIMAL = 0
+    EXPERIMENT = 1
+    TRIAL_NORM = 2
+    TRIAL_DETAILS = 3
```

Comment:
auto()?

Reply:
Since the int value is important (we want to be able to use `Verbosity.EXPERIMENT` and `int(1)` interchangeably, I'd rather specify them directly.